components/checkbox/index.css

```diff
@@ -131,25 +140,25 @@ governing permissions and limitations under the License.
 
 .spectrum-Checkbox-label {
   text-align: start;
-  margin-inline-start: var(--spectrum-checkbox-text-gap);
-  margin-block-start: var(--spectrum-checkbox-text-padding-top);
+  margin-inline-start: var(--spectrum-checkbox-text-to-control);
+  margin-block-start: var(--spectrum-checkbox-top-to-text);
 
-  font-size: var(--spectrum-checkbox-text-size);
-  font-weight: var(--spectrum-checkbox-text-font-weight);
-  font-style: var(--spectrum-checkbox-text-font-style);
-  line-height: var(--spectrum-checkbox-text-line-height);
+  font-size: var(--spectrum-checkbox-font-size);
+  /* font-weight: var(--spectrum-checkbox-text-font-weight); */
+  /* font-style: var(--spectrum-checkbox-text-font-style); */
```

Comment:
Waiting to hear back on what to do for `font-weight` and `font-style` values since they are not in the XD file.

Reply:
This is similar to how I notated these missing tokens in [my PR](https://github.com/adobe/spectrum-css/pull/1466). I also added a TODO comment line above them like so:

  /* TODO: font weight and style tokens */
  /* font-weight: var(--spectrum-radio-text-font-weight); **** missing token resolves to 400 ****/
  /* font-style: var(--spectrum-radio-text-font-style); **** missing token resolves to normal ****/

I am using _/* TODO */_ to mark any code that has an issue which still needs to be resolved.